.travis.yml

```diff
@@ -8,13 +12,17 @@ mono:
 
 solution: Dlr.sln
 
-before_install:
-  - wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.1/powershell_6.0.1-1.ubuntu.14.04_amd64.deb
-  - sudo dpkg -i powershell_6.0.1-1.ubuntu.14.04_amd64.deb
-
 sudo: required
 dist: trusty
 
+osx_image: xcode9.1
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then wget https://github.com/PowerShell/PowerShell/releases/download/v6.0.2/powershell_6.0.2-1.ubuntu.14.04_amd64.deb; fi
```

Comment:
We should probably use a wget command similar to what they use to get the dotnet sdk

```
wget --retry-connrefused --waitretry=1 -O /tmp/dotnet.pkg https://dotnetcli.azureedge.net/dotnet/Sdk/2.1.4/dotnet-sdk-2.1.4-osx-x64.pkg
```